layers/+lang/haskell/packages.el

```diff
@@ -45,10 +45,25 @@
     :if haskell-enable-ghc-mod-support
     :init (add-hook 'haskell-mode-hook 'ghc-init)
     :config
-    (when (configuration-layer/package-usedp 'flycheck)
-      ;; remove overlays from ghc-check.el if flycheck is enabled
-      (set-face-attribute 'ghc-face-error nil :underline nil)
-      (set-face-attribute 'ghc-face-warn nil :underline nil))))
+    (progn
+      (if (fboundp 'which-key-declare-prefixes)
+          (which-key-declare-prefixes
```

Comment:
Better to use `(spacemacs/declare-prefix-for-mode 'haskell-mode "mm" "haskell/ghc-mod")`.